contributors/design-proposals/service_links.md

```diff
@@ -0,0 +1,237 @@
+# Explicit service links
+
+## Abstract
+
+Proposes an API for explicit links to services in the pod spec.
+
+## Motivation
+
+Currently, all pods are automatically injected with docker-link-style
+environment variables for each service in their namespaces.  Making service
+dependencies explicit in the PodSpec has a number of advantages:
+
+1.  Reduce accidental coupling
```

Comment:
Very good point -- I think we need to have a pod-level notion of service dependencies.